### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by Pratik-Tech-Wizard**
## Description

This update implements a face recognition system using three models—CNN for feature extraction, SVM for classification, and ResNet for enhanced accuracy on suboptimal recognition rates. Key dependencies include TensorFlow, scikit-learn, and OpenCV for effective model training and image processing.

Fixes # (1582)

## Type of change

- [ ] Added a new machine learning frameworks, libraries or software.
- [x] Documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Pratik-Tech-Wizard** [comment]
@sanjay-kv Plz review and designate a appropriate label

**Review by sanjay-kv — APPROVED** [review]
